docs/en/modules/bloom.md

```diff
@@ -0,0 +1,103 @@
+# bloom Module
+| Since  | Origin / Contributor  | Maintainer  | Source  |
+| :----- | :-------------------- | :---------- | :------ |
+| 2017-11-13 | [Philip Gladstone](https://github.com/pjsg) | [Philip Gladstone](https://github.com/pjsg) | [bloom.c](../../../app/modules/bloom.c)|
+
+
+This module implements a bloom filter. This is a probabilistic data structure that is used to test for set membership. There are two operations -- `add` and `check` that allow 
+arbitrary strings to be added to the set or tested for set membership. Since this is a probabilistic dta structure, the answer returned can be incorrect. However,
+if the string *is* a member of the set, then the `check` operation will always return `true`. 
```

Comment:
Maybe add a link to https://en.wikipedia.org/wiki/Bloom_filter for further reading.